sql/moz-fx-cjms-nonprod-9a36/cjms_bigquery/flows_v1/metadata.yaml

```diff
@@ -30,3 +32,5 @@ bigquery:
     type: day
     field: submission_date
     require_partition_filter: false
+deprecated: true
```

Comment:
AFAIK deprecating these ETLs in the `moz-fx-cjms-nonprod-9a36` and `moz-fx-cjms-prod-f3c7` projects won't automatically remove access permissions like would happen for ETLs in the main `moz-fx-data-shared-prod` project, but it should still add a `deprecated: true` label to the tables and is the clearest way to indicate the intent here.